userguide/tutorials/payment_plugin.adoc

```diff
@@ -463,147 +475,148 @@ public PaymentTransactionInfoPlugin purchasePayment(UUID kbAccountId, UUID kbPay
+* Next, the GoCardless Mandate object is retrieved via `client.mandates().get(mandateId)` as explained in the https://developer.gocardless.com/api-reference/#mandates-get-a-single-mandate[_Gocardless documentation_].
 
-* Next, the `customerId` associated with the mandate is retrieved and all the `payments` associated with the customer are retrieved via `client.payments().all().withCustomer(customerId)` as explained https://developer.gocardless.com/api-reference/#payments-list-payments[here].
+* Next, the `customerId` associated with the mandate is retrieved and all the `payments` associated with the customer are retrieved via `client.payments().all().withCustomer(customerId)` as explained in the https://developer.gocardless.com/api-reference/#payments-list-payments[_Gocardless documentation_].
 
 * The code then iterates through the `payment` objects and obtains the `Payment` object corresponding to the `kbPaymentId` passed in. If you recall, the `purchasePayment` method sends `kbPaymentId` and `kbTransactionId` to GoCardless as metadata fields. Thus, the `kbPaymentId` is retrieved from the metadata of each `Payment` object and compared with the `kbPaymentId` passed in. 
 
-* The code then creates a `GoCardlessPaymentTransactionInfoPlugin` corresponding to a matching `Payment`using the following:
+* The code then creates a `GoCardlessPaymentTransactionInfoPlugin` corresponding to a matching `Payment` using the following:
 
-**  *killBillCurrency*  - The *GoCardless currency* (`com.gocardless.resources.Payment.Currency`) is retrieved from the `payment` object and converted to *Kill Bill currency* (`org.killbill.billing.catalog.api.Currency`).
+**  *payment amount*  - The *amount* is retrieved from the `payment` object.
 
+**  *killBillCurrency*  - The *GoCardless currency* (`com.gocardless.resources.Payment.Currency`) is retrieved from the `payment` object and converted to *Kill Bill currency* (`org.killbill.billing.catalog.api.Currency`).
 
 ** *status* - The *GoCardless status* (`com.gocardless.resources.Payment.Status`) is retrieved from the `payment` object and converted to *Kill Bill status* (`org.killbill.billing.payment.plugin.api.PaymentPluginStatus`).
 
 ** *kbTransactionId* - This is retrieved from the payment metadata. This step is very important because, the Janitor uses the *transaction id* to match the internal transactions with the plugin transactions as explained earlier. 
 
-** *outputProperties* - This is a List of `PluginProperty` objects. Some `PluginProperty` objects are created and added to the `outputProperties` List. Any arbitrary key-value pairs which need to be returned to the caller can be specified here. 
+** *outputProperties* - This is a List of `PluginProperty` objects and contains properties corresponding to the `mandateId` and `customerId. See <<plugin_property_notes, "Plugin Property Notes">> for more information.
```

Comment:
```suggestion
** *outputProperties* - This is a list of `PluginProperty` objects and contains properties corresponding to the `mandateId` and `customerId. See <<plugin_property_notes, Plugin Property Notes>> for more information.
```

Reply:
List is a Java interface, so L needs to be in uppercase. I've changed this to `List`